services/app/src/static/style/chess-board.css

```diff
@@ -2,18 +2,18 @@ table {
   text-align: center;
 }
 
-.white-tile,
-.black-tile {
+.border-tile,
+.non-border-tile {
```

Comment:
Naming is misleading here - I would expect `non-border-tile` to not have a border, but it does in this case.